doc/reference.md

```diff
@@ -361,22 +375,25 @@ to expect at that position:
 Links formats are [represented](#format-representations) as typed
 [references](#references) to other parts of the binary stream.
 
-| format               | `Repr` format               |
-| -------------------- | --------------------------- |
-| `link pos format`    | `Ref (Repr format)`         |
+| format            | `Repr` format |
+| ----------------- | ------------- |
+| `link pos format` | `Ref format`  |
 
-### Stream position formats
+### Deref formats
 
-The stream position format is interpreted as the current stream position during
-parsing:
+Deref formats allow [references](#references) to other parts of the stream to be
+parsed eagerly:
```

Comment:
I'm not sure if the eagerness of the parsing is the critical part of the semantics here, like the reason a spec author might need a deref is probably not most clearly expressed as eagerness exactly.

Reply:
Yeah I was worried when I wrote that and debated putting it in but was lazy (hah) - I'll try to fix it up tomorrow